core/src/main/resources/lib/hudson/help.jelly

```diff
@@ -33,10 +33,10 @@ THE SOFTWARE.
       The tooltip to display
     </st:attribute>
     <st:attribute name="iconSize">
-      Icon size, available are: small, medium, large, xlarge 
+      Icon size, available are: small, medium, large, xlarge
     </st:attribute>
   </st:documentation>
-  <a href="${href}">
-    <l:helpIcon iconSize="${iconSize}" class="icon-help" tooltip="${tooltip}"/>
+  <a href="${attrs.href}" class="jenkins-help-button ${attrs.iconSize}" tooltip="${attrs.tooltip}">
+    ?
```

Comment:
Looks really nice!
Maybe it could be possible to have that one available as jelly tag or to be the default over the round help icons in the future 👀

Reply:
Just finished a small follow up branch to do just that - thanks! :)

Reply:
I found a similar issue on the log recorder configuration page: 

![Screenshot from 2022-03-11 11-55-39](https://user-images.githubusercontent.com/985955/157854090-59b5dacf-5bb7-45ae-ace5-71716ee2b388.png)

but I cannot point the issue to a file change in this PR. Maybe it's my eyes, but I see the `?` icon to be different in size from the others I can find. 

I tested on Fedora with Firefox 97 using the `jenkins/core-pr-tester` Docker image.

Reply:
> Maybe it's my eyes, but I see the `?` icon to be different in size from the others I can find.

That's caused by the header size, I guess. If I change it from h1 to h4, it displays the typical help button like shown in other occasions:
![](https://i.imgur.com/phSA0Ww.png)

Reply:
fixed